### DISCUSSION TITLE
Albums/Shared Albums directories.

### DISCUSSION THREAD

**Discussion by RavMazurek** [immich-app/immich#6129] [General]
Hi guys,

I act as a company photographer - I share photos with them using shared link. I would love to create directory in there with <company name> that would act as top level album with direct link that would contain multiple album inside. It would be also cool to do with rest of the albums so I could create something like <family events> or <friends group hangouts> and have everything for everybody (or just me) in single place.

**Comment by bo0tzz** [comment]
Duplicate of #2073
